cmd/agent/Dockerfile

```diff
@@ -13,13 +13,9 @@ RUN apt-get update && apt-get install -qy libbpfcc-dev
 
 RUN make clean && make IMAGE_TAG=${IMAGE_TAG} RELEASE_BUILD=${RELEASE_BUILD} BUILD_IN_CONTAINER=false agent
 
-FROM debian:bullseye-slim
+FROM debian:bookworm-slim
```

Comment:
Is this a safe change?

Reply:
backed out, accidentally included a stale experiment.